src/components/UserItem/index.tsx

```diff
@@ -0,0 +1,128 @@
+import { t } from 'i18next';
+import React, { useState } from 'react';
+import { toast } from 'react-toastify';
+
+import { assets } from '@/assets/assets';
+import DriverAvatar from '@/components/DriverAvatar';
+import { COLORS } from '@/constants/colors';
+import { User } from '@/pages/TeamManagment/useFetchUsers';
+import UserForm from '@/pages/TeamManagment/UserForm/UserForm';
+import { deleteUser } from '@/services/usersService';
+import { Box, Divider, IconButton, Typography } from '@mui/material';
+
+import PopupMessage from '../PopupMessage';
+
+interface UserItemProps {
+  user: {
+    id: number;
+    fullName: string;
+    email: string;
+    phoneNumber: string;
+    role: string;
+  };
+  fetchUsers: () => void;
+  addUserToList: (user: User) => void;
+}
+
+const UserItem: React.FC<UserItemProps> = ({
+  user,
+  fetchUsers,
+  addUserToList,
+}) => {
+  const [isPopupOpen, setIsPopupOpen] = useState(false);
+  const firstName = user.fullName.split(' ')[0] || '';
+  const lastName = user.fullName.split(' ')[1] || '';
```

Comment:
create function in utility - getFirstName and getSeconName - assume it could be more then one place where this functionality could be needed